cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryStreamAdmin.java

```diff
@@ -105,7 +106,7 @@ public void drop(Id.Stream streamId) throws Exception {
   }
 
   @Override
-  public void register(Id.Stream streamId, Id.Program programId) {
-    usageRegistry.register(programId, streamId);
+  public void register(Id.Stream streamId, List<Id> owners) {
```

Comment:
nit: in some of the methods earlier, owners appears first, then stream. can we make it consistent?